Sources/TuistGraph/Graph/GraphOutputJSON.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+import TSCBasic
+
+public struct GraphOutputJSON: Codable, Equatable {
+    public var name: String
+    public var path: String
+    public var projects: [String: ProjectOutputJSON]
+    
+    public init(name: String, path: String, projects: [String: ProjectOutputJSON]) {
+        self.name = name
+        self.path = path
+        self.projects = projects
+    }
+    
+    public static func from(_ graph: Graph, withConfig config: Config) -> GraphOutputJSON {
+        var projects = [String: ProjectOutputJSON]()
```

Comment:
I think we should create methods for the individual models, such as:
```swift
public static func from(_ project: Project, with config: Config) -> ProjectOutput {...}
```

instead of putting everything into a single function.